components/sankey/overview.md

```diff
@@ -0,0 +1,108 @@
+---
+title: Overview
+page_title: Sankey Overview
+description: Overview of the Sankey diagram for Blazor.
+slug: sankey-overview
+tags: telerik,blazor,sankey,diagram,chart,overview
+published: True
+position: 0
+---
+@[template](/_contentTemplates/common/parameters-table-styles.md#table-layout)
+
+# Blazor Sankey Overview
+
+The <a href = "https://www.telerik.com/blazor-ui/sankey-diagram" target="_blank">Blazor Sankey diagram component</a> allows you to visualize changing flows and their distribution between domains. Sankey diagrams suit a variety of use cases like the representation of website traffic, budget breakdowns, energy flow, and others.
+
+## Creating Blazor Sankey Diagram
+
+1. Add the `<TelerikSankey>` tag to your razor page.
+1. [Bind the Sankey Data]({%slug sankey-data-binding%}).
+1. Set the [`ColorType` of the `SankeyLinks`]({%slug sankey-links%}) - if you choose `SankeyLinksColorType.Static`, also set the `Color` property of the `SankeyLinks`.
```

Comment:
The `Color` doesn't seem required, because the static color will be gray. We can simplify the step.

```suggestion
1. (Optional) Set the [`ColorType` of the `SankeyLinks`]({%slug sankey-links%}).
```